submissions/Nik3264/js-dom/index.js

```diff
@@ -0,0 +1,101 @@
+    },
+    {
+      header: "Deep Purple",
+      src: "./img/deep-purple.jpg",
+      text: "Deep Purple is recognized by The Guinness Book of World Records as the world’s “loudest band.” In 1972 the band played a show at the London Rainbow Theatre and rocked so hard that their sound reached an incredible 117dB. Three people were knocked unconscious as a result of the musical onslaught.",
+    },
+    {
+      header: "Pink Floyd",
+      src: "./img/Pink_Floyd_.jpg",
+      text: "The names Pink and Floyd came from two of Syd Barrett’s favourite Carolina bluesmen, Pink Anderson and Floyd Council, which he merged together and voila! People often assumed Pink Floyd was the name of a person in the band, especially stupid record industry people. The band sent this up with the line “which one’s Pink?” on the ‘Wish You Were Here’ song ‘Have A Cigar’ in 1975.",
+    },
+    {
+      header: "Scorpions",
+      src: "./img/scorpions.jpg",
+      text: "Scorpions are a German rock band formed in 1965 in Hanover by Rudolf Schenker. Since the band’s inception, its musical style has ranged from hard rock to heavy metal. The lineup from 1978–92 was the most successful incarnation of the group, and included Klaus Meine (vocals), Rudolf Schenker (rhythm guitar), Matthias Jabs (lead guitar), Francis Buchholz (bass guitar), and Herman Rarebell (drums).",
+    },
+  ];
+
+  const menuItem = document.querySelectorAll(".menu__item"),
+    menu = document.querySelector(".menu"),
+    contentItem = document.querySelector(".item");
+
+  function hideMenuItemsActive() {
+    menuItem.forEach((element) => {
+      element.classList.remove("menu__item__active");
+    });
+  }
+
+  function showMenuItemActive(i = 0) {
+    menuItem[i].classList.add("menu__item__active");
+  }
+
+  function renderNode(node, classItem, text, src) {
+    node.classList.add(classItem);
+    node.innerText = text;
+    node.src = src;
+    return node;
+  }
+
+  function renderContent(i = 0, parent) {
+    const header = document.createElement("h2"),
+      img = document.createElement("img"),
+      paragraph = document.createElement("p");
+
+    parent.innerHTML = "";
+    parent.appendChild(
+      renderNode(
+        (node = header),
+        (classItem = "item__header"),
+        (text = rockGroupsDescriptions[i].header)
+      )
+    );
+    parent.appendChild(
+      renderNode(
+        (node = img),
+        (classItem = "item__src"),
+        (text = ""),
+        (src = rockGroupsDescriptions[i].src)
+      )
+    );
+    parent.appendChild(
+      renderNode(
+        (node = paragraph),
+        (classItem = "item__text"),
+        (text = rockGroupsDescriptions[i].text)
+      )
+    );
+  }
+
+  hideMenuItemsActive();
+  showMenuItemActive(0);
+  renderContent(0, contentItem);
+
+  menu.addEventListener("click", (event) => {
+    event.preventDefault();
```

Comment:
-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
On mobile, when tapping on the menu, a blue highlight appeared, I have not noticed before. But now I've decided to remove it. The search for this bug was longer in time. I tried everything to somehow intercept the event and process it specifically on the target, and not affect the menu. But the solution turned out to be simple. Now I know it))). But of course, I forgot to remove everything unnecessary. It's because of the rush, I'm trying to move on as quickly as possible))